int32 OnOff_position
int32 OnOff_orientation
float64[3] goal_position
float64[4] goal_orientation
float64 goal_angle
float64 GoalmsTime
